examples/transpiler/babel.bzl

```diff
@@ -0,0 +1,37 @@
+"Adapter from the Babel CLI to the ts_project#transpiler interface"
+
+load("@npm//examples/@babel/cli:package_json.bzl", "bin")
+
+# buildifier: disable=function-docstring
+def babel(name, srcs, js_outs, map_outs, **kwargs):
+    # In this example we compile each file individually.
+    # You might instead use a single babel_cli call to compile
+    # a directory of sources into an output directory,
+    # but you'll need to:
+    # - make sure the input directory only contains files listed in srcs
+    # - make sure the js_outs are actually created in the expected path
+    for idx, src in enumerate(srcs):
+        # see https://babeljs.io/docs/en/babel-cli
+        args = [
+            "../../../../$(location %s)" % src,
+            "--presets=@babel/preset-typescript",
```

Comment:
hmm. if you did
```
--presets=examples/node_modules/@babel/preset-typescript
```
could you drop the chdir?

Reply:
maybe with a `./` on the front so nodejs doesnt try to resolve an npm package named "examples"
but do you think that's an improvement?

Reply:
although in its current shape I guess it would work as well... since the four `../` segments get you to the root

Reply:
execroot="../../../.."
...
        # see https://babeljs.io/docs/en/babel-cli
        args = [
            "{}/$(location {})".format(execroot, src),
            "--presets=@babel/preset-typescript",
            "--out-file",
            "{}/$(location {})".format(execroot, js_outs[idx]),
        ]

Reply:
maybe it would just be be a little clearer to read